src/QiskitCompletionProvider.ts

```diff
@@ -82,6 +82,10 @@ export class QiskitCompletionProvider implements ICompletionProvider {
     this.app = options.app;
   }
 
+  get timeout(): number {
+    return (this.settings.composite['providerTimeout'] as number) || 15000;
+  }
+
```

Comment:
(without checking the core code) is this overriding something? where is this called